src/researchhub_document/utils.py

```diff
@@ -130,24 +121,9 @@ def reset_unified_document_cache(
                                 f,
                                 time_scope,
                             ),
-                            priority=1,
-                            countdown=1
+                            priority=2,
```

Comment:
I lowered the priority a bit to (hopefully) avoid creating a bottleneck. @le-sun any thoughts on that?
Also, curious to know where `priority` is defined in the codebase since it seems to not be a native celery config